src/chttpd.erl

```diff
@@ -1068,3 +1057,18 @@ stack_hash(Stack) ->
 %% dedicated chunk.
 chunked_response_buffer_size() ->
     config:get_integer("httpd", "chunked_response_buffer", 1490).
+
+basic_headers(Req, Headers0) ->
+    Headers = Headers0
+        ++ server_header()
+        ++ couch_httpd_auth:cookie_auth_header(Req, Headers0),
+    chttpd_cors:headers(Req, Headers).
+
+handle_response(Req, Code, Headers, Args, Type) ->
+    couch_stats:increment_counter([couchdb, httpd_status_codes, Code]),
+    respond_(Req, Code, Headers, Args, Type).
+
+respond_(#httpd{mochi_req = MochiReq}, Code, Headers, _Args, start_response) ->
```

Comment:
Oh, no, as I said I'm indifferent here. I'd say I do share your opinion on naming function with numbers, but think both `respond_` and `do_respond` are acceptable, so it's really up to you to keep the name or rename it.